metro.config.js

```diff
@@ -1,11 +1,19 @@
-/**
- * Metro configuration for React Native
- * https://github.com/facebook/react-native
- *
- * @format
- */
+// eslint-disable-next-line import/no-extraneous-dependencies
+const blacklist = require('metro-config/src/defaults/blacklist');
+
+// blacklist is a function that takes an array of regexes and combines
```

Comment:
I suggest we start doing something like this: https://www.cnn.com/2020/07/03/tech/twitter-jpmorgan-slave-master-coding/index.html

Ex. `Denylist` instead of `Blacklist`